### PR TITLE
Added Simple Get Route for Incidents

### PR DESCRIPTION
**Pull request by RyEggGit**
Added a route for Epic 3 to get the organization's incidents
 
_Also, I figured out a way to type the ORM models. I imported [ `from __future__ import annotations` ](https://stackoverflow.com/questions/61544854/from-future-import-annotations) at the top of the files to complete the generic with a type instead of a class. Also, to remove the errors when creating a new orm model manually, if you add `def __init__(self, **kwargs): super().__init__(**kwargs),` they magically go away, and you get fully type support._

### CODE REVIEW

**Review by DMalone87 — APPROVED** [review]
Looks great!

**Comment by DMalone87** [comment]
Sorry; I missed that this was still a draft. Please wait for @RyEggGit 's confirmation to merge.

**Comment by RyEggGit** [comment]
wat

> Sorry; I missed that this was still a draft. Please wait for @RyEggGit 's confirmation to merge.

Sorry @DMalone87 had to make some changes after talking to @hrauniya. I just added a privacy setting to the incidents for if the org wants to have their incidents private/public and added some more tests. After review should be good to merge.

File: backend/database/models/incident.py
Comment (DMalone87) [inline backend/database/models/incident.py:80]: Let's call this `privacy_filter`. `source_type` is a bit misleading.

**Pull request merged**
